src/NUnitEngine/Addins/nunit.v2.driver/NUnit2FrameworkDriver.cs

```diff
@@ -68,12 +66,28 @@ public NUnit2FrameworkDriver(AppDomain testDomain)
 
             var initializer = DomainInitializer.CreateInstance(_testDomain);
             initializer.InitializeDomain((int)InternalTrace.Level);
-
-            _runner = RemoteTestRunner.CreateInstance(_testDomain, Process.GetCurrentProcess().Id);
         }
 
         public string ID { get; set; }
 
+        private TestRunner Runner {
+            get
+            {
+                if (_runner == null)
+                {
+                    int runnerId;
+                    if (ID == null || !int.TryParse(ID, out runnerId))
+                    {
+                        throw new InvalidSuiteException("ID must be defined at first as a string representation of an integer value.");
```

Comment:
Also, can we add something to the message indicating that it's the V2 driver that requires an int?